bang/magicollections.py

```diff
@@ -1897,6 +1897,24 @@ def to_fields(self, view, item, *args, **kwargs):
         setSubField(fields, 'length', key='value', value=lambda f: item.length_in_minutes)
         setSubField(fields, 'unlock', key='value', value=item.unlock_sentence)
 
+        for difficulty, verbose_name in models.Song.DIFFICULTIES:
+            note_image = u'{}img/note.png'.format(RAW_CONTEXT['static_url'])
+            image = lambda f: u'{static_url}img/songs/{difficulty}.png'.format(
+                static_url=RAW_CONTEXT['static_url'],
+                difficulty=difficulty,
+            )
+            diff = getattr(item, u'{}_difficulty'.format(difficulty), None)            
+            setSubField(fields, u'{}_notes'.format(difficulty), key='image', value=image)            
+            if diff != None:
```

Comment:
When comparing with `None`, it's better to use `is` and `is not`:

`!=` -> `is not`

Reply:
✅